events/class_tests/event_tests.py

```diff
@@ -58,3 +57,34 @@ def test_exist_event(self):
     def test_invalidation(self, title, date_time_start, date_time_end, expected_error):
         with pytest.raises(ValidationError, match=expected_error):
             create_event(title, date_time_start, date_time_end).save()
+
+    def test_get_all_meetings(self):
+        result = Event.objects.get_all_meetings()
+        for title_id in ['1', '3']:
+            assert Event.objects.get(title=f"event{title_id}") in result
+        assert result.count() == 2
+
+    def test_get_all_user_meetings(self, user_id=1):
+        result = Event.objects.get_all_user_meetings(user_id)
+        for title_id in ['1', '3']:
+            assert Event.objects.get(title=f"event{title_id}") in result
+        assert Event.objects.get(title="event2") not in result
+        assert result.count() == 2
+
+    @pytest.mark.parametrize('event_title, user_id, year, month', [
+        ("event1", 1, 2020, 3),
+        ("event3", 1, 2020, 1)
+    ])
+    def test_get_all_user_month_meetings(self, event_title, user_id, year, month):
+        assert Event.objects.get(title=event_title) in Event.objects.get_all_user_month_meetings(user_id, year, month)
+
+    @pytest.mark.parametrize('event_title, user_id, date', [
+        ("event1", 1, datetime(2022, 3, 24)),
+        ("event3", 1, datetime(2022, 1, 24))
+    ])
```

Comment:
please add ids.
for more details you can look at:
https://docs.pytest.org/en/6.2.x/example/parametrize.html#different-options-for-test-ids

and make sure you have ids in all the other parameterize

Reply:
done